proposals/0002.md

```diff
@@ -0,0 +1,318 @@
+* Title: Structured error handling in Rust
+* Champion: @CodeSandwich
+* Conversation (optional): [thread on radicle.community](https://radicle.community/t/proposal-structured-error-handling-in-rust/179)
+* References:
+  * [The previous error handling proposal](https://github.com/radicle-dev/radicle-decisions/blob/master/proposals/0001.md)
+  * [`std::error::Error`](https://doc.rust-lang.org/std/error/trait.Error.html)
+  * [A discussion about the good design of errors](https://users.rust-lang.org/t/do-people-not-care-about-printable-error-chains-a-k-a-how-to-nicely-implement-display-for-an-error/35362)
+
+# Summary/Motivation/Introduction
```

Comment:
Can we choose one? Intro seems appropriate.

Reply:
I didn't understand the template intention. Fixed.